core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -964,6 +965,14 @@ public PropServerConfiguration(
                     ilpTcpWorkerCount = 6;
                 }
```

Comment:
Assuming enough cpus , default for io pool is higher than for writers so maybe prioritising number of dedicated io workers would make more sense ?